kolibri/core/content/api.py

```diff
@@ -291,6 +295,17 @@ def filter_available(self, queryset, name, value):
         return queryset.filter(root__available=value)
 
 
+class ChannelThumbnailView(View):
+    def get(self, request, channel_id):
+        channel = get_object_or_404(models.ChannelMetadata, id=channel_id)
+        try:
+            mimetype, b_64_thumbnail = channel.thumbnail.split(",", 1)
```

Comment:
In manual testing, I noticed that this is giving a mimetype of `data:image/png;base64` for example.

Adding:
```
# Split the mimetype to remove data: prefix and ;base64 suffix.
mimetype = mimetype.split(":", 1)[1].split(";", 1)[0]
```
as another line within the try except block, strips it down to just the mimetype.

It doesn't seem to make any difference to rendering, but seems neater to have the mimetype reported precisely.

Reply:
makes sense, updated!